src/issues/components/IssueHistoryItem.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div>
+    <QItem
+      multiline
+      class="clickable"
+      :class="{'greyed': detailIsShown}"
+      @click.native="toggleDetail"
+    >
+      <QItemMain>
+        <QItemTile>
+          {{ $t('ISSUE.VOTING.RESULTS.KEEP_DISCUSSING') }}
+        </QItemTile>
+        <QItemTile
+          stamp
+          class="mobile-only text-weight-light"
+        >
+          <DateAsWords :date="issue.createdAt" />
+        </QItemTile>
+      </QItemMain>
+      <QItemSide
+        class="desktop-only"
+        stamp
+        right
+      >
+        <DateAsWords :date="issue.createdAt" />
+      </QItemSide>
+    </QItem>
+    <Transition name="slide-toggle">
```

Comment:
I'd keep them here because in case one issue has many votings the page would be unecessarily long and cluttered if all details were immediately shown. Also I think the transitions are quite fancy...^^

Reply:
It's only about the transitions really. Not the biggest fan of them myself.

The collapse seems fine, could be made into a QCollapsible though.